examples/css2d_label.html

```diff
@@ -134,10 +135,8 @@
 
 				requestAnimationFrame( animate );
 
-				var elapsed = clock.getElapsedTime();
-
-				moon.position.set( Math.sin( elapsed ) * 5, 0, Math.cos( elapsed ) * 5 );
-
+				moon.position.applyAxisAngle(new THREE.Vector3(0,2,0),Math.PI/120);
```

Comment:
>new THREE.Vector3(0,2,0)

Avoid doing this. You are instantiating a new vector every frame unnecessarily. Create one and reuse it.

Reply:
Thank you for your comment